helm/opendistro-es/files/configurer/configurer.sh

```diff
@@ -160,11 +160,13 @@ create_role_definitions() {
   echo "Creating role definitions"
 
   create_role() {
+    echo "Role name: $1"
+    echo "role_definition: $2"
     role_name=$1
     role_definition=$2
     response=$(curl -X PUT "${es_url}/_opendistro/_security/api/roles/${role_name}" \
       -H 'Content-Type: application/json' \
-      --insecure --silent \
+      -k -s \
```

Comment:
In my opinion it's fine to use the longer version of the flags when writing scripts to make it more readable. That way someone who doesn't know what `-k` or `-s` does wont have to look it up every time they forget it.
Talking from experience :grin: